src/pages/Location.js

```diff
@@ -2,36 +2,67 @@ import React from 'react';
 import '../static/css/Location.css';
 import PropTypes from 'prop-types';
 import ParallaxComponent from '../components/Parallax/ParallaxComponent';
+import useMediaQuery from '../hooks/useMediaQuery';
 
 const Location = (props) => {
   const { pageIndex } = props;
-  return (
-    <div className="location-background">
-      <div className="split left">
-        <div className="location-body">
-          <ParallaxComponent
-            pageIndex={pageIndex}
-            transitionTime={1500}
-            transitionDelay={0}
-            styles={{ height: '100vh' }}
-          >
-            <p className="help-location-text">
-              The Ray Welcome Center <br />
-            </p>
-            <p className="help-location-text2">
-              The Ray is a proving ground for the evolving ideas and technologies that will transform the transportation
-              infrastructure of the future, and it starts on 18 miles of West Georgia&apos;s I-85, and the land and
-              communities surrounding it.
-            </p>
-          </ParallaxComponent>
+  const isPageMobile = useMediaQuery('(max-width: 850px)');
+
+  const locationPage = () => {
+    if (isPageMobile) {
+      return (
+        <ParallaxComponent pageIndex={pageIndex} transitionTime={1500} transitionDelay={0} styles={{ height: '100vh' }}>
```

Comment:
<img width="214" alt="Screenshot 2022-09-12 at 5 50 46 PM" src="https://user-images.githubusercontent.com/25470781/189783685-dd0e508b-ac9a-4787-ae6f-7756dc32f64d.png">
Looks good but text needs to be left aligned and if possible, could you add in the location pointer for the map?

Reply:
Hey Edmund, getting the location pointer will be a bit of challenge so currently it just uses an arrow on the image to point to the location as shown below. 

![image](https://user-images.githubusercontent.com/55326650/190021367-3690281a-eca4-4ff1-bf8a-6035baf9ea7f.png)